mkdocs.yml

```diff
@@ -33,7 +31,8 @@ nav:
           - About our Decision Process & Terminology: guides/genomics-platform/requesting-data/glossary.md
       - Managing Data:
           - Managing Data Overview: guides/genomics-platform/managing-data/working-with-our-data.md
-          - Uploading/Downloading Data: guides/genomics-platform/managing-data/data-transfer-app.md
```

Comment:
Did you check to make sure there were no other direct references to the DTA page embedded in other guides?

Reply:
I have not yet, but I will.